forge-gui/res/cardsfolder/upcoming/dino_dna.txt

```diff
@@ -0,0 +1,7 @@
+Name:Dino DNA
+ManaCost:1
+Types:Artifact
+A:AB$ ChangeZone | PrecostDesc$ Imprint — | SorcerySpeed$ True | Cost$ 1 T | Imprint$ True | Origin$ Graveyard | Destination$ Exile | ValidTgts$ Creature | SpellDescription$ Exile target creature card from a graveyard. Activate only as a sorcery.
+A:AB$ CopyPermanent | Cost$ 6 | Defined$ Targeted | SetPower$ 6 | AddKeywords$ Trample | SetToughness$ 6 | SetColor$ Green | SetCreatureTypes$ Dinosaur | SorcerySpeed$ True | ValidTgts$ Creature.IsImprinted+ExiledWithSource | TgtZone$ Exile | TgtPrompt$ Select target creature card exiled with Dino DNA | SpellDescription$ Create a token that's a copy of target creature card exiled with CARDNAME, except it's a 6/6 green Dinosaur creature with trample. Activate only as a sorcery.
```

Comment:
Is there a case where this card's name could change and then the TgtPrompt would be incorrect?

Might just need to introduce the CARDNAME functionality into TgtPrompt - these similar cards might all be impacted
![image](https://github.com/Card-Forge/forge/assets/103371817/a0210c7e-2527-401b-9ff0-98720c9d49eb)